webpack.config.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/* eslint-env node */
+
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+const { bundler } = require( '@ckeditor/ckeditor5-dev-utils' );
+const UglifyJsWebpackPlugin = require( 'uglifyjs-webpack-plugin' );
+
+module.exports = {
+	context: __dirname,
+
+	devtool: 'source-map',
+	performance: { hints: false },
+
+	entry: path.join( __dirname, 'src', 'ckeditor.jsx' ),
+
+	output: {
+		library: 'CKEditor',
+
+		path: path.join( __dirname, 'dist' ),
+		filename: 'ckeditor.js',
+		libraryTarget: 'umd',
+		libraryExport: 'default',
+
+	},
+
+	optimization: {
+		minimizer: [
+			new UglifyJsWebpackPlugin( {
+				sourceMap: true,
+				uglifyOptions: {
+					output: {
+						// Preserve CKEditor 5 license comments.
+						comments: /^!/
+					}
+				}
+			} )
+		]
+	},
+
+	plugins: [
+		new webpack.BannerPlugin( {
```

Comment:
IMO we shouldn't do that. That build contains the whole react library.

Reply:
And thus I'd add React to the extrnals to `webpack.config.js` when building our react component.

Reply:
It'll also prevent loading React few times.

Reply:
Done.